misc/test/examples_test.go

```diff
@@ -243,23 +243,24 @@ func TestExamples(t *testing.T) {
 				"aws:region": awsRegion,
 			},
 		}),
-		base.With(integration.ProgramTestOptions{
-			Dir: path.Join(cwd, "..", "..", "aws-ts-ruby-on-rails"),
-			Config: map[string]string{
-				"aws:region":     awsRegion,
-				"dbUser":         "testUser",
-				"dbPassword":     "2@Password@2",
-				"dbRootPassword": "2@Password@2",
-			},
-			ExtraRuntimeValidation: func(t *testing.T, stack integration.RuntimeValidationStackInfo) {
-				// Due to setup time on the vm this output does not show up for several minutes so
-				// increase wait time a bit
-				maxWait := 20 * time.Minute
-				assertHTTPResultWithRetry(t, stack.Outputs["websiteURL"], nil, maxWait, func(body string) bool {
-					return assert.Contains(t, body, "New Note")
-				})
-			},
-		}),
+		// See https://github.com/pulumi/examples/pull/366 for details on why this was disabled.
```

Comment:
Instead of linking to the PR, could you open 2 issues (one for each failing test) and add `TODO[examples#NNN]: Fix failing test` comments for each?

Reply:
Sure thing.

Reply:
Done

Reply:
Created #367 and #368.